modules/ROOT/pages/set-dlb.adoc

```diff
@@ -0,0 +1,123 @@
+= Set a Dedicated Load Balancer URL for Anypoint Datagraph
+
+If you’re unable to use Port 8082 on a CloudHub shared load balancer, you must set a URL for Anypoint DataGraph to use on a dedicated load balancer (DLB).
+
+You set the URL using a DLB mapping rule that you add to the Anypoint DataGraph configuration.
+
+=== Example Mapping Rules for a DLB URL
+
+Based on the way DLB mapping rules work, you must configure the *Input Path* (Input URI) and the *Output Path* (App URI) so that the final DLB URL is able to access the endpoint of the Anypoint DataGraph application, which is `/graphql`.  
+
+For example, consider the following mapping rules for `examplelb.lb.anypointdns.net` depending on the value of the *Input Path* and a Datagraph app name of `datagraph-xz082edx-1585-901xe374-d0x0`: 
+
+[%header%autowidth.spread]
+|===
+|Index |Input URI (Input Path) |App name |App URI (Output Path) | Protocol
+|0 |`/graphql` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/graphql` |HTTPS
+|1 |`/datagraph/` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/graphql` |HTTPS
+|2 |`/datagraph/` |`datagraph-xz082edx-1585-901xe374-d0x0` |`/` |HTTPS
+|3 |`/{app}/` |`{app}` | `/` |HTTPS
+|===
+
+For each index, the appropriate DLB URL is:
+
+* Index 0: `examplelb.lb.anypointdns.net/graphql`
+* Index 1: `examplelb.lb.anypointdns.net/graphql`
```

Comment:
Url for index 1 must be:

Index 1: `examplelb.lb.anypointdns.net/datagraph/`

Reply:
made that change